client/src/components/NavBar/style.ts

```diff
@@ -1,7 +1,7 @@
 import { Classes } from "@blueprintjs/core";
 import styled from "@emotion/styled";
 import { css } from "@emotion/react";
-import { Chip, CommonThemeProps, fontCapsXxxs } from "czifui";
+import { Tag, CommonThemeProps, fontCapsXxxs } from "@czi-sds/components";
```

Comment:
no more "<Chip> will be deprecated..." console warning 🎉

Reply:
HURRAY!! You the beast!